components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -19,14 +19,26 @@ import (
 	storage "github.com/chef/automate/components/authz-service/storage/v2"
 	"github.com/chef/automate/components/authz-service/testhelpers"
 	"github.com/chef/automate/lib/grpc/auth_context"
-	uuid "github.com/chef/automate/lib/uuid4"
 )
 
 const (
 	Applied = "applied"
 	Staged  = "staged"
 )
 
+const (
```

Comment:
THANK YOU <3

Reply:
I'm afraid I didn't catch them all, but just spend some time on it and left off when I felt I had caught enough... 😉 There's still a few queries inline.